modules/ROOT/pages/servicenow/servicenow-reference-6.6.0.adoc

```diff
@@ -0,0 +1,413 @@
+= ServiceNow Connector Reference - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+ServiceNow Connector Version 6.6
+
+Anypoint Connector for ServiceNow (ServiceNow Connector) provides connections between Mule runtime engine (Mule) and ServiceNow apps.
+
+Release Notes: xref:release-notes::connector/servicenow-connector-release-notes-mule-4.adoc[ServiceNow Connector Release Notes]
+
+
+== Configurations
+
+[[config]]
+=== Config
+
+
+==== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+|Name | String | The name for this configuration. Connectors reference the configuration with this name. | | x
+| Connection a| * <<config_basic-connection, Basic Authentication>> 
+* <<config_oauth2-auth-code-connection, OAuth 2.0 Authorization Code>> 
+ | The connection types to provide to this configuration. | | x
+| Expiration Policy a| <<ExpirationPolicy>> |  Configures the minimum amount of time that a dynamic configuration instance can remain idle before the runtime considers it eligible for expiration. This does not mean that the platform expires the instance at the exact moment that it becomes eligible. The runtime purges the instances when as appropriate. |  | 
+|===
+
+==== Connection Types
+[[config_basic-connection]]
+===== Basic Authentication
+
+
```

Comment:
Again, not sure about these empty sections. I'm sure you know what you are doing; just mentioning it.